transcom/mymove
# B-21581: Incorrect GBLOC on Move When Origin Duty Location is Updated

## [Agility Ticket B-21581](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21581)

## Testing Steps I took
Use [this doc](https://dp3.atlassian.net/wiki/download/attachments/2279735297/postal_code_to_gblocs_202311091157.csv?api=v2) to determine what zip codes to use in order to see different GBLOCs.
1. As a service member (to include CIV or other), create a move with a shipment. Pay special attention to what selection is made for your origin duty location.
2. As SC and/or TOO verify which GBLOC shows up in the Origin GBLOC column for the move when you search for it. Also verify that when you open the move as an office user, the GBLOC should also show in the top-right of the page as you are looking at the move/shipment.
3. As either the service member or as an office user, change the origin duty location to that which falls under a different GBLOC (reference the document link above which is from the helpful files section of the ByteSize M&Ms Confluence pages).
4. As SC and/or TOO verify that the GBLOC has changed.
